### DISCUSSION TITLE
Inconsistent labelling of content status between list view and form view

### DISCUSSION THREAD

**Discussion by jmurth1234** [payloadcms/payload#5985] [Feature Requests & Ideas]
### Describe the Bug

We've noticed a minor issue with the labelling of post status being inconsistent on the list view and the form view. If you have a published document that has a draft version after the publish date it is marked as draft

<img width="1141" alt="image" src="https://github.com/payloadcms/payload/assets/581104/623260d4-ca5b-4637-9814-cb4e36214e64">

However on the list view itself it appears as changed

<img width="463" alt="image" src="https://github.com/payloadcms/payload/assets/581104/dc7e60a2-871c-47ac-9a24-161574692d19">

Should this not be consistent? It makes it dificult for cms editors to know what content is truly published 

### To Reproduce

1. Take published item
2. Make edit
3. Observe inconsistency

### Payload Version

2.13.0

**Comment by DanRibbens** [comment]
@rymate1234,

Thanks for reporting your issue. I've moved it to a discussion based because it represents new functionality beyond how things currently work.

I agree that this would be confusing, in that you only see the latest status and it would be helpful to know if it there is a version that is published, not just the most recent status. For the list view to work the collection is queried with drafts=true so that it can show published and non-published records. 

To add to the user experience we could add a virtual field to versions that can show separately from the `status`. This is just one idea, I'm open to others so long as we aren't making a breaking change.

**Comment by wkentdag** [comment]
I was able to implement a "fix" on my own through hooks + custom components:

1. Create a hook where you determine whether a document is a true draft vs "changed" (published doc w/ pending draft updates -- [here's how I do it](https://github.com/wkentdag/payload-plugin-algolia/blob/main/src/hooks/syncWithSearch.ts#L42-L68)), and assign that value to a virtual field
2. Create a custom `Cell` component where you modify the UI to distinguish between the two cases. eg, [here](https://github.com/wkentdag/payload-plugin-scheduler/blob/main/src/fields/PublishDate/components/Cell.tsx) I change text color + add an icon to differentiate "scheduled" drafts from regular ones.

I do think this would be nice to have in core tho, since the default draft behavior is to serve the latest published version even if there's newer drafts, and the list view UI would make you think otherwise. Could be as simple as changing the Cell text from "Draft" to "Published (drafts pending)" or similar 💭
